tool/tctl/common/token_command.go

```diff
@@ -103,14 +103,15 @@ func (c *TokenCommand) Initialize(app *kingpin.Application, config *service.Conf
 	c.tokenAdd.Flag("db-name", "Name of the database to add").StringVar(&c.dbName)
 	c.tokenAdd.Flag("db-protocol", fmt.Sprintf("Database protocol to use. Supported are: %v", defaults.DatabaseProtocols)).StringVar(&c.dbProtocol)
 	c.tokenAdd.Flag("db-uri", "Address the database is reachable at").StringVar(&c.dbURI)
+	c.tokenAdd.Flag("format", "Output format, 'text', 'json' or 'token'").Default(teleport.Text).StringVar(&c.format)
 
 	// "tctl tokens rm ..."
 	c.tokenDel = tokens.Command("rm", "Delete/revoke an invitation token").Alias("del")
 	c.tokenDel.Arg("token", "Token to delete").StringVar(&c.value)
 
 	// "tctl tokens ls"
 	c.tokenList = tokens.Command("ls", "List node and user invitation tokens")
-	c.tokenList.Flag("format", "Output format, 'text' or 'json'").Hidden().Default(teleport.Text).StringVar(&c.format)
+	c.tokenList.Flag("format", "Output format, 'text' or 'json'").Default(teleport.Text).StringVar(&c.format)
```

Comment:
Should the help message here say "text, json, or token" as well?

Also, you may want to use `.EnumVar()` which will get you some extra validation for free.

Reply:
That string is in the help menu, I belive `Flag()` adds it by default.

Good catch, I'll do that.